buku

```diff
@@ -2307,12 +2306,11 @@ class BukuDb:
         for item in sublist:
             if item['type'] == 'folder':
                 next_folder_name = folder_name + DELIM + strip_delim(item['name'])
-                for i in self.traverse_bm_folder(
+                yield from self.traverse_bm_folder(
                         item['children'],
                         unique_tag,
                         next_folder_name,
-                        add_parent_folder_as_tag):
-                    yield i
+                        add_parent_folder_as_tag)
```

Comment:
Same as in the other two pull-requests, these changes are necessary to pass the lint.